_sass/hallios/_masthead.scss

```diff
@@ -4,12 +4,20 @@
 
 .masthead {
   position: relative;
-  border-bottom: 1px solid $border-color;
   -webkit-animation: $intro-transition;
   animation: $intro-transition;
   -webkit-animation-delay: 0.15s;
   animation-delay: 0.15s;
   z-index: 20;
+  @if variable-exists(border-color) {
+    border-bottom: 1px solid $border-color;
+  }
+  @if variable-exists(header-border-color) {
+    border-bottom: $header-border-color;
+  }
+  @if variable-exists(header-background-color) {
+    background-color: $header-background-color;
+  }
```

Comment:
These ifs ensure that even if the theme doesn't have the variable, it won't crash the app